aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +261,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Forms in layers vs. full page 
+
+
+- Single column form layout creates a clear focused path to completion for the user.
+- Multi-column layouts can cause users to skip important fields, input data in a wrong field, or simply hinder efficiency which leads to frustration or even abandonment. 
+- Display no more than 10 form fields with in the same viewport so that users are not intimidated with seeing a long page filled with form fields and sections.
```

Comment:
Do we have a solution/recommendation we can suggest for use cases where there are a large number of fields? Should we point them to Wizard?

"Consider presenting the [form in a wizard](/templates/wizards) to break the form in to digestible, focussed sections."

Reply:
This was updated